libs/app-utils/models/src/lib/app-error.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export type AppErrorType = string
+
+export class AppError<
+  Type extends AppErrorType = AppErrorType,
+  Payload = unknown,
+> extends Error {
+  override message: string
```

Comment:
Error class already has the `message` field so why override it? Shouldn't this be enough?

```TSX
export class AppError<
  Type extends AppErrorType = AppErrorType,
  Payload = unknown,
> extends Error {
  constructor(
    public readonly type: Type = "UnknownError" as Type,
    message = "",
    public readonly payload?: Payload
  ) {
    super(message)

    if (Error.captureStackTrace) {
      Error.captureStackTrace(this, AppError)
    }
  }
```